ChangeLog.md

```diff
@@ -1,7 +1,8 @@
 ## Changes Between 2.11.0 and 2.12.0 (unreleased)
 
-No changes yet.
+Added support to manage user limits.
```

Comment:
This does not follow the formatting convention used in earlier releases. When in doubt, don't touch the change log at all.